components/txn_types/src/types.rs

```diff
@@ -253,6 +253,32 @@ pub enum MutationType {
     Other,
 }
 
+/// A row mutation.
+#[derive(Debug, Clone)]
+pub enum RawMutation {
+    /// Put `Value` into `Key` with TTL. The TTL will overwrite the existing TTL value.
+    Put {
+        key: Key,
+        value: Value,
+        ttl: Option<u64>,
```

Comment:
Since `ttl==0` means the key will never expire, why not just use `u64`?

Reply:
if tikv enable_ttl option is false, the value is None

Reply:
I think it's better to make `enable_ttl ` a field of AtomicStore.